src/__tests__/AdvancedSearch.test.tsx

```diff
@@ -83,48 +52,46 @@ describe("renders advanced search correctly", () => {
   });
 
   test("Hides languages when no languages are passed", () => {
-    render(
-      <AdvancedSearch
-        searchQuery={SearchQueryDefaults}
-        languages={{ status: "200", data: [] }}
-      />
-    );
+    render(<AdvancedSearch languages={{ status: "200", data: [] }} />);
     expect(
       screen.queryByRole("group", { name: "Languages" })
     ).not.toBeInTheDocument();
   });
-
-  test("Prepopulates form input based on search query parameters", () => {
-    render(
-      <AdvancedSearch
-        searchQuery={complicatedSearchQuery}
-        languages={defaultLanguages}
-      />
-    );
-
-    expect(screen.getByLabelText("english")).toBeChecked();
-    expect(screen.getByLabelText("From")).toHaveValue(1990);
-    expect(screen.getByLabelText("To")).toHaveValue(1999);
-    expect(screen.getByLabelText("Keyword")).toHaveValue("cat");
-  });
 });
 
 describe("Advanced Search submit", () => {
   test("Submits well formed query", () => {
     render(
       <MockNextRouterContextProvider>
-        <AdvancedSearch
-          searchQuery={complicatedSearchQuery}
-          languages={defaultLanguages}
-        />
+        <AdvancedSearch languages={defaultLanguages} />
       </MockNextRouterContextProvider>
     );
+
+    const inputValues = {
+      Keyword: "cat",
+      Author: "Nook",
+      Subject: "poetry",
+      Title: "Handbook",
+    };
+    inputTerms.forEach((val) => {
+      fireEvent.change(screen.getByLabelText(val.label), {
+        target: { value: inputValues[val.label] },
+      });
+    });
+
+    userEvent.click(screen.getByRole("checkbox", { name: "english" }));
+    fireEvent.change(screen.getByRole("spinbutton", { name: "From" }), {
```

Comment:
Looks like we test for year, but not for all the other filters (format, language) - can we just make a change to all of them and make sure they all work?

Reply:
Yep! I must've missed the format.